### ISSUE TITLE
Drop Ruby

### ISSUE THREAD

**Issue by hyandell** [todogroup/repolinter#286]
## Issue Report

Licensee and Linguist are used as axioms to allow some linting rules to only apply when the output of those tools match the license or language. Unfortunately, both are written in Ruby which doubles the tech-stack complexity of the project. I'm not aware of anyone actively using these axioms and so my proposal is that we drop these.

**Comment by justinabrahms** [comment]
When I run repolinter, I pay attention to the axiom gated results.

We could probably do just as well as linguist (for my needs at least) by looking at file extensions.

**Comment by hyandell** [comment]
Agreed. https://www.npmjs.com/package/linguist-js is also an option. Sounds like it's built on the rules for github linguist, but not the ruby code.

Licensee wise, losing the axiom would lose the apache-notice rule. While not having it available would also affect the license-detectable-by-licensee rule; though that is off by default.

**Comment by Brend-Smits** [comment]
Losing the license check would be a shame, have to see if we can find a replacement for this package. 
It would be fantastic if the axioms could just be given their own home in a new repository, and we can just use them plugin-style. I understand that this might be challenging with the current architecture.
